doc/barman.1.d/50-list-backups.md

```diff
@@ -1,9 +1,13 @@
 list-backups *SERVER_NAME*
 :   Show available backups for `SERVER_NAME`. This command is useful to
-    retrieve a backup ID. For example:
+    retrieve a backup ID and the backup type. For example:
 
 ```
-servername 20111104T102647 - Fri Nov  4 10:26:48 2011 - Size: 17.0 MiB - WAL Size: 100 B
+servername 20111104T102647 - F - Fri Nov  4 10:26:48 2011 - Size: 17.0 MiB - WAL Size: 100 B
 ```
 
-    In this case, *20111104T102647* is the backup ID.
+In this case, *20111104T102647* is the backup ID, and `F` is the backup type label for a full backup taken with `pg_basebackup`. The backup type label displayed by this command takes one of the following values:
```

Comment:
```suggestion
In this case, *20111104T102647* is the backup ID, and `F` is the backup type label for a full backup taken with `pg_basebackup`. The backup type label displayed by this command uses one of the following values:
```
I'm honestly not sure that `takes` is the right word there... `uses` is possibly clearer/more correct?